src/tsx/App.tsx

```diff
@@ -1,17 +1,16 @@
-import Faq from "./components/sections/Faq";
-import Introduction from "./components/sections/Introduction";
-import Process from "./components/sections/Process";
-import Stakeholder from "./components/sections/Stakeholder";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import HomePage from "./pages/HomePage";
+import ProjectPage from "./pages/ProjectPage";
 
-function App() {
+export default function App() {
   return (
-    <div>
-        <Introduction />
-        <Process />
-        <Stakeholder />
-        <Faq />
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/">
+          <Route index element={<HomePage />} />
+          <Route path="projekt" element={<ProjectPage />} />
+        </Route>
+      </Routes>
+    </BrowserRouter>
   );
 }
-
-export default App;
```

Comment:
I prefer this syntax before you make any changes (`export default ...` at the bottom), but it's up to you.